Packages/KeyAppKit/Sources/KeyAppNetworking/HTTPClient/HTTPResponseDecoder.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+/// Decoder for response from `HTTPClient`
+public protocol HTTPResponseDecoder {
+    func decode<T: Decodable>(_ type: T.Type, data: Data, httpURLResponse response: HTTPURLResponse) throws -> T
+}
+
+/// ResponseDecoder for JSON type
+public class JSONResponseDecoder: HTTPResponseDecoder {
```

Comment:
add final to class

Reply:
also I see that `JsonRpcDecoder ` is a struct, can you change `JSONResponseDecoder ` to a struct also?

Reply:
Yes. Done. Thanks